is08/outputs.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2018 British Broadcasting Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from is08.action import Action
+from is08.helperTools import getIOList
+from GenericTest import NMOSTestException
+from is08.testConfig import globalConfig
+from is08.calls import Call
+
+
+def getOutputList():
+    outputList = getIOList("output")
+    instanceList = []
+    for outputId in outputList:
+        instanceList.append(ACMOutput(outputId))
+    return instanceList
+
+
+class ACMOutput:
+
+    def __init__(self, outputID):
+        self.id = outputID
+        self.url = "{}outputs/{}/".format(globalConfig.apiUrl, self.id)
+        self.test = globalConfig.test
+
+    def assembleOutputObject(self):
+        """Create JSON representation of an Output"""
+        toReturn = {}
+        resourceList = {
+            'sourceid': 'source_id',
+            'channels': 'channels',
+            'caps': 'caps',
+            'properties': 'properties'
+        }
+        for apiId, ioId in resourceList.items():
+            url = "{}{}".format(self.url, apiId)
+            if apiId == 'sourceid':
+                call = Call(url, True)
```

Comment:
In case the `source_id` is null, this leads to an error. 
In this case the json `null` is interpreted as a string, in the case of the getIO-object-pasing fort the test_01, it is (correctly) threatened as a json `null`. 
This leads in test 1 to a comparison of `None` != `"null"`.